integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/bundle/BundledReadAlarmRegister.feature

```diff
@@ -7,14 +7,25 @@ Feature: SmartMetering Bundle - ReadAlarmRegister
   As a grid operator 
   I want to be able to read the alarm register from a meter via a bundle request
 
-  Background: 
+  Scenario Outline: Retrieve read alarm register response of a device in a bundle request for <protocol> <version>
     Given a dlms device
-      | DeviceIdentification | TEST1024000000001 |
-      | DeviceType           | SMART_METER_E     |
-
-  Scenario: Retrieve events of a device in a bundle request
-    Given a bundle request
+      | DeviceIdentification     | TEST1024000000001 |
+      | DeviceType               | SMART_METER_E     |
+      | SelectiveAccessSupported | true              |
+      | Protocol                 | <protocol>        |
+      | ProtocolVersion          | <version>         |
+      | Port                     | <port>            |
+      | Lls1active               | <lls1active>      |
+      | Hls5active               | <hls5active>      |
+    And a bundle request
       | DeviceIdentification | TEST1024000000001 |
     And the bundle request contains a read alarm register action
     When the bundle request is received
     Then the bundle response should contain a read alarm register response
+    Examples:
+      | protocol | version | port | lls1active | hls5active |
+      | DSMR     | 2.2     | 1026 | true       | false      |
```

Comment:
Volgens mij is de nieuwe conventie dat we de poort niet meer specificeren, deze wordt automatcisch bepaald op basis van het protocol

Reply:
correct, port is er uit gehaald.